charts/datasaur/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.8.1
+version: 1.9.0
```

Comment:
this should be a patch update not a minor update @nangirvan

Reply:
@daniel-jeremia updated